### ISSUE TITLE
QA

### ISSUE THREAD

**Issue by gabitoesmiapodo** [BootNodeDev/dAppBoosterLanding-v1#5]
This thing needs to be tested by an actual person or persons.

Also some suggestions to improve titles, descriptions, order for the items in the demo list, etc. would be good.

**Comment by jose-garrofe** [comment]
Looks great so far.
On a quick test run done today I could only find two issues that are absolutely minor:
- Hash Component: Nice to have. View in explorer could open the active chain explorer instead of always using etherscan.
- Token Input: Balance check works too slow on Op. chain.

We can do another smoke test run once the development is completed.

**Comment by gabitoesmiapodo** [comment]
@jose-garrofe thanks for the review.

- The hash component should work according to your request now.
- I'll ask @fernandomg about Token Input's balance issue.

Note: all the demos were moved to https://github.com/BootNodeDev/dAppBoosterLandingPage

Preview (main branch) https://dappboosterlandingpage.vercel.app/

**Comment by fernandomg** [comment]
> I'll ask @fernandomg about Token Input's balance issue.

I'll work on that

**Comment by jose-garrofe** [comment]
@gabitoesmiapodo 
Hash component retested. Works perfect. Thanks!

**Comment by tloffler** [comment]
I was tasked with organizing the built-in features and refining their descriptions. Additionally, I made some design adjustments:

- Added a search box.
- I organized the features in a three-column grid, but to highlight the more critical ones or for layout reasons, some are displayed in a two-column grid.
- Removed feature icons to simplify the design.
- Eliminated white spaces and reduced font size.
- Prioritized the demo block, placing it before the title.

Here’s the [Figma link](https://www.figma.com/proto/h9vA5Xy4GPyb4e1JbeldHP/Starter-kit?page-id=1034%3A1759&node-id=1685-2864&node-type=FRAME&viewport=-7349%2C-6112%2C0.77&t=FQvEGWsN1C2Rp8P6-1&scaling=min-zoom&content-scaling=fixed). Thoughts?
